<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Imię:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelClass">
        <property name="text">
         <string>Klasa:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBoxClass">
        <item><property name="text"><string>Wojownik</string></property></item>
        <item><property name="text"><string>Barbarzyńca</string></property></item>
        <item><property name="text"><string>Bard</string></property></item>
        <item><property name="text"><string>Druid</string></property></item>
        <item><property name="text"><string>Mag</string></property></item>
        <item><property name="text"><string>Czarnoksiężnik</string></property></item>
        <item><property name="text"><string>Zaklinacz</string></property></item>
        <item><property name="text"><string>Łowca</string></property></item>
        <item><property name="text"><string>Łotr</string></property></item>
        <item><property name="text"><string>Mnich</string></property></item>
        <item><property name="text"><string>Kleryk</string></property></item>
        <item><property name="text"><string>Paladyn</string></property></item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLevel">
        <property name="text">
         <string>Poziom:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxLevel">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelAge">
        <property name="text">
         <string>Wiek:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxAge">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelGender">
        <property name="text">
         <string>Płeć:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxGender">
        <item><property name="text"><string>Mężczyzna (M)</string></property></item>
        <item><property name="text"><string>Kobieta (K)</string></property></item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelRace">
        <property name="text">
         <string>Rasa:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QComboBox" name="comboBoxRace">
        <item><property name="text"><string>Człowiek</string></property></item>
        <item><property name="text"><string>Elf</string></property></item>
        <item><property name="text"><string>Półork</string></property></item>
        <item><property name="text"><string>Półelf</string></property></item>
        <item><property name="text"><string>Gnom</string></property></item>
        <item><property name="text"><string>Niziołek</string></property></item>
        <item><property name="text"><string>Krasnolud</string></property></item>
        <item><property name="text"><string>Diabelstwo</string></property></item>
        <item><property name="text"><string>Drakon</string></property></item>
        <item><property name="text"><string>Mroczny elf</string></property></item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelAlignment">
        <property name="text">
         <string>Światopogląd:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboBoxAlignment">
        <item><property name="text"><string>Praworządny Dobry</string></property></item>
        <item><property name="text"><string>Neutralny Dobry</string></property></item>
        <item><property name="text"><string>Chaotyczny Dobry</string></property></item>
        <item><property name="text"><string>Praworządny Neutralny</string></property></item>
        <item><property name="text"><string>Neutralny</string></property></item>
        <item><property name="text"><string>Chaotyczny Neutralny</string></property></item>
        <item><property name="text"><string>Praworządny Zły</string></property></item>
        <item><property name="text"><string>Neutralny Zły</string></property></item>
        <item><property name="text"><string>Chaotyczny Zły</string></property></item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="labelBackground">
        <property name="text">
         <string>Tło fabularne:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QTextEdit" name="textEditBackground"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutEquipment">
        <item>
         <widget class="QLabel" name="labelEquipment">
          <property name="text">
           <string>Ekwipunek</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetEquipment"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutAddEquipment">
          <item>
           <widget class="QLineEdit" name="lineEditAddEquipment"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAddEquipment">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRemoveEquipment">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutSkills">
        <item>
         <widget class="QLabel" name="labelSkills">
          <property name="text">
           <string>Umiejętności</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetSkills"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutAddSkill">
          <item>
           <widget class="QLineEdit" name="lineEditAddSkill"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAddSkill">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRemoveSkill">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="pushButtonNew">
        <property name="text">
         <string>Nowa postać</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOpen">
        <property name="text">
         <string>Otwórz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Zapisz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>